src/transformers/models/maskformer/feature_extraction_maskformer.py

```diff
@@ -484,12 +484,13 @@ def post_process_panoptic_segmentation(
                 thing. If not set, defaults to the `is_thing_map` of COCO panoptic.
 
         Returns:
-            `List[Dict]`: A list of dictionaries, one per image, each dictionary containing two keys:
+            `List[Optional[Dict]]`: A list of dictionaries, one per image, each dictionary containing two keys:
```

Comment:
Maybe breaking this, but if I send 2 images and I receive only 1 there's no way to know which one it was that had the segmentation.

We can't return an empty segmentation since there's an index error in the code (didn't look thoroughly on this, I assumed the flag `detect_something` was necessary).

@sgugger @FrancescoSaverioZuppichini

Reply:
Nice catch, it's true. If we don't detect, then the output will just be an empty `List`

Reply:
Not necessary anymore, you change was better than this one.